pootle/static/js/editor/components/UnitSource.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import { t } from 'utils/i18n';
+
+
+const UnitSource = React.createClass({
+
+  propTypes: {
+    id: React.PropTypes.number.isRequired,
+    values: React.PropTypes.array.isRequired,
+    hasPlurals: React.PropTypes.bool.isRequired,
+    sourceLocaleCode: React.PropTypes.string,
+    sourceLocaleDir: React.PropTypes.string,
+  },
+
+  getUnitClass() {
+    const pluralClass = this.props.values.length > 1 ? " translate-plural" : '';
```

Comment:
Thanks. There is a method `unit.hasplural()` in backend. So I'll replace it with `this.props.hasPlurals`.